user/applications/tinker/src/burnin_test.cpp

```diff
@@ -568,25 +579,40 @@ bool BurninTest::testGnss() {
 
     // Enable GNSS. It can take some time after the modem AT interface comes up for the GNSS engine to start
     const int RETRIES = 10;
-    int r = 0;
-    for (int i = 0; i < RETRIES && r != RESP_OK; i++) {
-        r = Cellular.command("AT+QGPS=1\r\n");
+    
+    bool success = false; 
+    for (int i = 0; i < RETRIES && !success; i++) {
```

Comment:
This loop is used elsewhere as well.  It would be handy to have a macro/template that wraps retry loops with delay.  Don't worry about it today.